src/lib/pages/backoffice/Loan/LoanDetails/AvailableItems/AvailableItems.js

```diff
@@ -221,6 +259,31 @@ export default class AvailableItems extends Component {
         >
           <Loader isLoading={isLoading}>
             <Error error={error}>{this.renderTable()}</Error>
+            <Modal open={modalOpen}>
+              <Modal.Header>Overbooked literature!</Modal.Header>
+              <Modal.Content>
+                The{' '}
```

Comment:
I would change the message, because if I am the librarian:
* I have the book in my hands, I know which book it is
* I want to know which one is the first loan request that I should fulfil
* It might actually not be overbooked, the GitHub issue was wrong!

The link should open **in a new tab**:
```
<Modal.Header>Antecedent loan request found!</Modal.Header>
<Modal.Content>There is an antecedent <link>pending loan request</link> for this literature. Are you sure that you want to checkout the copy for this request?</Modal.Content>
```

I don't like the `antecedent` word, but I cannot find anything else. We should say that there is a "previous" loan request created...